client/Pages/EventTypeCreate/Models.elm

```diff
@@ -79,6 +79,15 @@ defaultRetentionDays =
     4
 
 
+defaultEnvelope : Bool -> String
```

Comment:
There is nothing "default" stored there. it is a pure function, I would call it just `boolToString` helper function. I would even move it to Helper/String.elm near `stringToBool` as it commonly used https://github.com/zalando-nakadi/nakadi-ui/blob/0e6a4fda60ac2b93755decfe7c83c0f5aa5f8716/client/Helpers/JsonEditor.elm#L328